app/components/validated-input.js

```diff
@@ -0,0 +1,44 @@
+// Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies
+
+// BEGIN-SNIPPET validated-input
```

Comment:
This seems odd. I did see some `v-get` errors in ember-osf, but:
1. Other validation fields still showed up and worked even if v-get didn't, and
2. Max is reporting that validation mostly works in his project. (not developing inside an addon)

I'd encourage us to make sure this is necessary in the actual use context, before including this in the final PR.